### ISSUE TITLE
Skip Navigation -- quick tab/enter option

### ISSUE THREAD

**Issue by bmcutler** [Submitty/Submitty#3912]
**What problem are you trying to solve with Submitty**
keyboard/screenreader users often want to quickly skip the sidebar (esp when it contains > 10 menu items) and jump quickly into the interesting items within the content portion of the page.

**Describe the way you'd like to solve this problem**
Let's make the first item on the side bar be 'skip navigation'.
But let's have it be hidden/not visible, EXCEPT when it has focus. 
When it has focus, it should be a visible option, probably overlaid on top of the side bar, towards the top.  

When you first load the page it's not visible (nothing on the page has focus yet).
When you hit tab the first time, the skip navigation now has focus, so it's visible.  
If you hit tab a second time, focus moves to the first thing on the side bar, skip navigation disappears again.
Or instead if you have focus on skip navigation and press enter, focus moves to the first content link within the page.

**Comment by bmcutler** [comment]
(written up after meeting discussions)
tagging @pbailie @mcglow2
